tests/general/InvalidLinkBearTest.py

```diff
@@ -141,3 +144,29 @@ def test_redirect_threshold(self):
 
         self.assertResult(valid_file=long_url_redirect,
                           invalid_file=short_url_redirect)
+
+    def test_ignore_regex(self):
+
+        ignored_URLs = """
+        http://sub.example.com
+        http://sub.example.com/something
+        """.splitlines()
+
+        not_ignored_URLs = """
+        http://www.example404.com
+        http://exampe.com
+        http://example.co.in
```

Comment:
yes

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala-analyzer.org/ - http://viperdev.io/ - http://gitmate.com/

On 26 May 2016 at 20:07, Sanket Dasgupta <notifications@github.com> wrote:

> In tests/general/InvalidLinkBearTest.py
> <https://github.com/coala-analyzer/coala-bears/pull/458#discussion_r64792765>
> :
>
> > @@ -141,3 +144,29 @@ def test_redirect_threshold(self):
> >
> >          self.assertResult(valid_file=long_url_redirect,
> >                            invalid_file=short_url_redirect)
> > +
> > +    def test_ignore_regex(self):
> > +
> > +        ignored_URLs = """
> > +        http://sub.example.com
> > +        http://sub.example.com/something
> > +        """.splitlines()
> > +
> > +        not_ignored_URLs = """
> > +        http://www.example404.com
> > +        http://exampe.com
> > +        http://example.co.in
>
> oops really sorry. but goodexample.com returns 200, so how about
> 404example.com?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/458/files/f9a83c43db67283008f95b75099df0cb00d8935c..e84956bf449be3a82b79fc1d8c46fc5ffbc1f8f1#r64792765>
>